webservices/args.py

```diff
@@ -632,7 +644,11 @@ def _deserialize(self, value, attr, data):
         IStr(validate=validate.OneOf(['S', 'O'])),
         description='Support or opposition',
     ),
-    'is_notice': fields.List(fields.Bool, description='Record filed as 24- or 48-hour notice'),
+    'min_expenditure_date': fields.Date(description='Selects all items expended by this committee after this date'),
+    'max_expenditure_date': fields.Date(description='Selects all items expended by this committee before this date'),
+    'min_expenditure_amount': fields.Date(description='Selects all items expended by this committee greater than this amount'),
+    'max_expenditure_amount': fields.Date(description='Selects all items expended by this committee less than this date'),
```

Comment:
Would it make sense to move these descriptions into the `docs` module?  Also, should we remove the commented out lines?